e2e_tests/tests/cluster/test_webhooks.py

```diff
@@ -22,7 +22,7 @@ class WebhookRequestHandler(SimpleHTTPRequestHandler):
     def do_POST(self) -> None:
         global request_to_webhook_endpoint
         global keep_server_running
-        content_length = int(self.headers.get("content-length"))
+        content_length = int(self.headers.get("content-length", 0))
```

Comment:
Did this fail before when the get request failed? Looks like default used to be `None`. Not sure if this change will cover up potential regressions. Maybe have someone from backend review?

Reply:
Oh, I didn't notice that backend already signed off, it's probably fine then. Feel free to mark this as resolved. :)